_includes/article-categories.html

```diff
@@ -3,6 +3,9 @@
     <h6 class="font-weight-bold text-uppercase text-muted mb-0">Tags:</h6>
   </div>
   <div class="col ml-n5">
+    <a class="badge badge-pill badge-secondary-soft" href="/articles">
```

Comment:
```suggestion
    <a class="badge badge-pill badge-secondary-soft" href="/articles/">
```

Always add a `/` at the end since CloudFront redirects always to a `/` which if we add it always dose one redirect less to the user.

Reply:
Aha, no problem. I wasn't aware of CloudFront being involved. That would explain a previous bug! Thanks.